rasa/nlu/extractors/LookupEntityExtractor.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import os
+from typing import Any, Dict, List, Optional, Text
+
+import rasa.utils.common as common_utils
+from rasa.constants import DOCS_URL_COMPONENTS
+from rasa.nlu import utils
+from rasa.nlu.components import Component
+from rasa.nlu.config import RasaNLUModelConfig
+from rasa.nlu.constants import ENTITIES
+from rasa.nlu.extractors.extractor import EntityExtractor
+from rasa.nlu.model import Metadata
+from rasa.nlu.training_data import Message
+
+logger = logging.getLogger(__name__)
+
+
+class LookupEntityExtractor(EntityExtractor):
+    """
+    Searches for entities in the user's message from a list of examples.\n
```

Comment:
I've noticed the `\n` character appear in the docstrings. Is there a reason for this?

Reply:
I have added it so that whenever I see the class definition it would look properly formatted

![image](https://user-images.githubusercontent.com/49223934/87157688-3c63f980-c2dc-11ea-8401-d3fe303df835.png)

Let me know if I need to remove the formatting with "\n"

Reply:
Every editor is bound to render it differently, but it's probably best to stick to the PEP guidelines here which is to remove the `\n`.